src/nft/components/bag/BagFooter.tsx

```diff
@@ -218,7 +218,7 @@ const InputCurrencyValue = ({
     )
   }
 
-  if (tradeState === TradeState.LOADING) {
+  if (tradeState === TradeState.LOADING && !trade) {
```

Comment:
Will there ever be a valid case where `tradeState === TradeState.LOADING && trade`? If not, this could remain as-is to be simpler/easier-to-read.

Reply:
So, for "syncing" state, we're not going to have any UI response? BTW, would be great to add a comment